pages/common/sort.md

```diff
@@ -13,3 +13,7 @@
 - Sort passwd file by the 3rd field
 
 `sort -t: -k 3n /etc/passwd`
+
+- Sort file containing sizes in human readable form
+
+`sort -h {{filename}}`
```

Comment:
File containing sizes? This is something artificial.
Maybe (I do not insist) in this case it is better to use pipe:

```md
`ls -lh | sort -h`
```